Src/GoogleApis.Auth.DotNet4/Properties/AssemblyInfo.cs

```diff
@@ -28,8 +28,14 @@
 [assembly: AssemblyCopyright("Copyright © Google Inc 2013")]
 [assembly: AssemblyTrademark("")]
 [assembly: AssemblyCulture("")]
-// Google.Apis.Auth.DotNet4.Tests won't compile in "ReleaseSigned" configuration.
-#if DEBUG
+
+#if SIGNED
+[assembly: InternalsVisibleTo("Google.Apis.Auth.DotNet4.Tests,PublicKey=" +

```

Comment:
Just curious. What is this key? I see also that you use the same one in the gRPC project.
Besides that LGTM...  Thanks for doing that!